libs/elements/cards/src/lib/components/elewa-price-list/elewa-price-list.component.scss

```diff
@@ -0,0 +1,46 @@
+.card-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(280px, 1fr));
+  gap: 20px;
+}
+
+/* Target the Angular component using attribute selector */
+[elewa-website-price-item-card] {
+  margin-bottom: 10px;
+}
```

Comment:
don't do this. only use html tags,classes or ids to set styles in your stylesheet